iteration.qmd

```diff
@@ -285,9 +290,18 @@ expand_dates <- function(df) {
       )
     )
 }
+
+df2 <- tibble(
+  name = c("Amy", "Bob"),
+  date = as.Date(c("2009-08-03", "2010-01-16"))
+)
+
+df2 |> 
+  expand_dates()
+# I think deserves to be applied to some actual data, i.e. complete the example. Most readers won't be able to imagine what the `list()` thing does. And I guess my immediate attempt to use it has failed?
```

Comment:
I think this example should be taken to its logical conclusion, i.e. `expand_dates()` should be shown "in action".